src/LoadOptionsDialog.cpp

```diff
@@ -138,14 +139,22 @@ void LoadOptionsDialog::addFiles() {
     "*.sr2 *.srf *.srw "
     "*.x3f"
     ")"));
-    QStringList files = QFileDialog::getOpenFileNames(this, tr("Open raw images"),
-        lastDirSetting.isNull() ? QDir::currentPath() : QDir(lastDirSetting.toString()).absolutePath(),
-        filter, Q_NULLPTR, Q_NULLPTR);
-    if (!files.empty()) {
-        QString lastDir = QFileInfo(files.front()).absolutePath();
-        settings.setValue("lastOpenDirectory", lastDir);
-        for (auto & i : files) {
-            new FileItem(i, fileList);
+
+
+    QFileDialog loadDialog(this, tr("Open raw images"), lastDirSetting.isNull() ? QDir::currentPath() : QDir(lastDirSetting.toString()).absolutePath(), tr("Digital Negatives (*.dng)"));
```

Comment:
This line chaned HDRMerge into DNG-onlyMerge ;)
It needs the `filter` from above to allow opening raw files other than DNG.

Reply:
Damn, copypasta error :disappointed: Will be fixed in next commit.